package.json

```diff
@@ -144,7 +144,7 @@
     "tslint-etc": "1.9.2",
     "tslint-no-toplevel-property-access": "0.0.2",
     "tslint-no-unused-expression-chai": "0.0.3",
-    "typescript": "~3.8.3",
+    "typescript": "^3.9.2",
```

Comment:
This needs to remain a tilde semver range. It should not be changed to a caret range.

Reply:
@cartant made the changes please have a look